packages/plugin-headphone-check/src/index.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { startTimeline } from "@jspsych/test-utils";
+
+import jsPsychHeadphoneCheck from ".";
+
+jest.useFakeTimers();
+
+describe.skip("headphone-check plugin", () => {
+  it("should load", async () => {
+    const { expectFinished, getHTML, getData, displayElement, jsPsych } = await startTimeline([
+      {
+        type: jsPsychHeadphoneCheck,
```

Comment:
Maybe a quick update here?

Reply:
I created a dummy test, but I can't for the life of me figure out how to mock the `AudioPlayer` class...

Reply:
I mocked the AudioPlayer class in https://github.com/jspsych/jsPsych/pull/3179 and some use cases are shown in the commits here https://github.com/jspsych/jsPsych/pull/3179/commits/d296962c7ea761ed64e7ed7e35974b8151109b7d